corehq/apps/domain/forms.py

```diff
@@ -1290,13 +1290,17 @@ def __init__(self, domain, web_user, *args, **kwargs):
 
     @property
     def form_actions(self):
-        return FormActions(
-            crispy.ButtonHolder(
-                crispy.Submit(
-                    self.slug,
-                    ugettext_noop('Update')
+        return (
+            crispy.Hidden('slug', self.slug),
+            FormActions(
+                crispy.ButtonHolder(
+                    crispy.Submit(
+                        self.slug,
+                        ugettext_noop('Update'),
```

Comment:
I think this can be a regular old `ugettext`. I don't think this will be translated in this case.